infrastructure/lambda/private-api.yaml

```diff
@@ -171,6 +214,13 @@ components:
       properties:
         session_id:
           type: "string"
+    PersonInfo:
```

Comment:
@ElliotMurphyGDS Does this section need to be included int her Auth Schema?

Reply:
No personInfo only needs session